src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -242,6 +242,7 @@ private static void UpdateTelemetry(Stopwatch sw, ApiEvent apiEvent, Authenticat
             authenticationResult.AuthenticationResultMetadata.DurationInCacheInMs = apiEvent.DurationInCacheInMs;
             authenticationResult.AuthenticationResultMetadata.TokenEndpoint = apiEvent.TokenEndpoint;
             authenticationResult.AuthenticationResultMetadata.CacheRefreshReason = apiEvent.CacheInfo;
+            authenticationResult.AuthenticationResultMetadata.BrokerTelemetry = apiEvent.WamTelemetry;
```

Comment:
Nit, but the `BrokerTelemetry` naming suggests generic broker use, while `WamTelemetry` is Windows-specific. Is this intentional?

Reply:
+1 a more generic name would serve us better long term

Reply:
This is internal. I'll change it to "runtime telemetry".